pkgs/tools/virtualization/awsebcli/default.nix

```diff
@@ -0,0 +1,89 @@
+ { stdenv, python }:
+let
+  inherit (localPythonPackageSet) pkgs;
+  inherit (pkgs) fetchPypi buildPythonApplication;
+
+  localPythonPackageSet = python.override {
+    packageOverrides = self: super: rec {
+      docker = super.docker.overridePythonAttrs (oldAttrs: rec {
+          pname = "docker-py";
```

Comment:
you can use oldAttrs

Reply:
Unfortunately the name of the Pypi package changed in 2.0: https://github.com/docker/docker-py/releases/tag/2.0.0

`docker-py` is the name of the package prior to that but the `pname` from `oldAttrs` is `docker`.

Unless you're saying to use `${oldAttrs.pname}-py`?